Source/Python/application/main.py

```diff
@@ -51,13 +54,22 @@ def __init__(
         traveler_repository = repositories_factory.traveler_repo
         event_repository = repositories_factory.event_repo
 
-        location_use_case = LocationUseCase(location_repository, event_repository)
-        traveler_use_case = TravelerUseCase(traveler_repository, event_repository)
-        event_use_case = EventUseCase(location_repository, traveler_repository, event_repository)
-        timeline_use_case = TimelineUseCase(location_repository, traveler_repository, event_repository)
+        self._location_use_case = LocationUseCase(location_repository, event_repository)
+        self._traveler_use_case = TravelerUseCase(traveler_repository, event_repository)
+        self._event_use_case = EventUseCase(location_repository, traveler_repository, event_repository)
+        self._timeline_use_case = TimelineUseCase(location_repository, traveler_repository, event_repository)
+
+    def initialize_controllers(self, *, rest_controller_config: dict) -> None:
+        rest_controller = RESTControllersFactory(**rest_controller_config).rest_controller
 
         request_handlers_factory = RequestHandlersFactory(
-            location_use_case, traveler_use_case, event_use_case, timeline_use_case, **request_handlers_config)
+            rest_controller,
+            self._location_use_case, self._traveler_use_case, self._event_use_case, self._timeline_use_case)
         self._locations_request_handler = request_handlers_factory.location_handler
         self._travelers_request_handler = request_handlers_factory.traveler_handler
         self._event_request_handler = request_handlers_factory.event_handler
+
+        self._locations_request_handler.register_routes()
+        self._travelers_request_handler.register_routes()
```

Comment:
... Why is there not a call to the events request handler?

Reply:
The Events handler route registration was being done in the init. Improved these to be a static method.